setup.py

```diff
@@ -48,7 +60,12 @@
     include_package_data=True,
     author_email='',
     license='MIT',
-    install_requires=install_requires,
+    python_requires='>=3.7',
+    install_requires=requirements,
+    extras_requires={
+        'cubit': extra_requirements_cubit,
```

Comment:
Does this work? Usually you can't use git packages as extra
iirc pypi will complain

Reply:
@eduardogr

Reply:
Thanks for pointing that, I'm checking that...

Reply:
removed till we know how we want to add this extra